test/e2e/functional_test.go

```diff
@@ -641,6 +641,9 @@ spec:
 }
 
 func (s *FunctionalSuite) TestStorageQuotaLimit() {
+	// TODO Test fails due to unstable PVC creation and termination in K3S
```

Comment:
PVC creation is behaving inconsistently in k3s. I am investigating in another PR.